theia-extensions/viewer-prototype/src/browser/trace-viewer/trace-viewer.tsx

```diff
@@ -241,6 +241,14 @@ export class TraceViewerWidget extends ReactWidget {
             });
     }
 
+    storeState(): OutputDescriptor[] {
+        return this.outputDescriptors;
```

Comment:
Just to let you know, the calling method is using JSON.stringify(). The data structure OutputDescriptor defines a 2 optional fields `start` and `end` which are of type `bigint`. The JSON.stringify() will throw an exception because it can't serialize bigints. 

With the current Trace Compass server back-end `start` and `end` is never sent and hence there are not there when storeState() is called. Hence you won't see this exception. 

I'm currently debating with myself, whether we merge this PR as is and address the serialization issue later on, or we handle it right away. I'll let you know. Please still fix the comment on line 248 below right way.